scripts/gpu_support/nvidia/install_cuda_and_libraries.sh

```diff
@@ -239,9 +239,9 @@ for EASYSTACK_FILE in ${TOPDIR}/easystacks/eessi-*CUDA*.yml; do
       echo_green "all installations at ${EESSI_SITE_SOFTWARE_PATH}/software/... succeeded!"
     fi
 
-    # clean up tmpdir
-    rm -rf "${tmpdir}"
-
     # Restore MODULEPATH for next loop iteration
     MODULEPATH=${SAVE_MODULEPATH}
 done
+
+# clean up tmpdir
+rm -rf "${tmpdir}"
```

Comment:
We can do a final clean-up here. However, shouldn't we also do a clean-up at the end of processing an easystack file? I think we may want to either create the tmpdir at the beginning of the loop (and remove it completely at the end) _OR_ remove the contents of the directory, e.g., (original line 243)
```bash
rm -rf "${tmpdir}"/*
```